webapp/exercise_admin/templates/exercise_admin/file-upload-exercise-change.html

```diff
@@ -330,4 +147,12 @@
   <div display="none">
     {% test_tab True True True %}
   </div>
+  <div display="none">
+    <table>
+      {% include_file_tr True True %}
+    </table>
+  </div>
+  <div display="none">
+    {% include_file_popup True True True %}
+  </div>
```

Comment:
Change all to `style="display: none;"` instead. That existing one is probably an accident.